components/dashboard/src/onboarding/signup-goals.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export const SIGNUP_GOALS_OTHER = "other";
+
+export const getSignupGoalsOptions = () => {
+    return [
+        {
+            label: "Replace remote/containerized development (VDI, VM based, Docker Desktop..)",
+            value: "replace_remote_containerized_dev",
+        },
+        { label: "More powerful dev resources", value: "powerful_dev_resources" },
+        { label: "Just exploring CDEs", value: "exploring_cdes" },
```

Comment:
Maybe make it (`CEDs`) possible to add a link here?

Reply:
@mustard-mh I think, He meant Cloud Dev Environments ([CDEs](https://www.gitpod.io/cde))

Reply:
A link would be great - going to leave that for a followup as I'd need to adjust a few things with how these labels render so it supports it.

Reply:
> @mustard-mh I think, He meant Cloud Dev Environments ([CDEs](https://www.gitpod.io/cde))

Yep, I know it, but for users may don't know this shorthand new word 😂 just want to give them more context